libs/build_condaenv.sh

```diff
@@ -26,6 +26,10 @@ if $MODULES; then
   set +x
   source $MODULESHOME/init/bash
   module load hpc-$HPC_PYTHON
+  if [[ -n "${STACK_miniconda3_version:-} " ]]; then
+    module is-loaded python  && module unload python
+    module load miniconda3/${STACK_miniconda3_version:-}
+  fi
```

Comment:
You shouldn't need to do this between lines 29-32.

The correct way would be to define `HPC_PYTHON=miniconda3/miniconda3_version` in your `config.sh` file.  
When `setup_modules.sh` is executed, it will create a `hpc-miniconda3` modulefile.
Then L28, `module load hpc-$HPC_PYTHON` will load the correct `miniconda3/miniconda3_version` module.